liquibase-dist/src/main/archive/examples/xml/liquibase.sqlplus.conf

```diff
@@ -21,7 +21,10 @@
 ####
 
 # The full path to the SQLPLUS executable.
+# Sample linux path
 # liquibase.sqlplus.path=/apps/app/12.2.0.1.0/oracle/product/12.2.0.1.0/client_1/bin/sqlplus
+# Sample windows path
+# liquibase.sqlplus.path=c:\oracle\product\11.2.0\client_1\bin\sqlplus.exe
```

Comment:
This example needs to change to `\\` for the Windows path
For instance 
`# liquibase.sqlplus.path=c:\\oracle\\product\\11.2.0\\client_1\\bin\\sqlplus.exe`